### DISCUSSION TITLE
Ищем самое краткое описание продукта

### DISCUSSION THREAD

**Discussion by sergeifedorenko** [easably/FindMyLevel#1] [Ideas]
Как варинт
Check a text/video for getting the complexity of text for those who learn English

**Comment by I-am-Artyom** [comment]
Name options: LevelYoText or LevelText, LevelCheck

Description: Know the complexity of your English text
Get the level of English used in texts/videos

**Reply by sergeifedorenko** [reply]
Со всем согласен но просто check мало
Это как кнопка на сайте агенства недвижимости "продать мою квартиру" или "максимально оценить мою квартиру"

**Comment by lllana** [comment]
> Как варинт
> Check a text/video for getting the complexity of text for those who learn English

Check a text & video complexity, for English learners.
